README.md

```diff
@@ -214,6 +214,29 @@ yarn workspace @asap-hub/contentful ctf-migrate create add-foo-field -c bar
 
 Note: the `content_type` is the slug, not the display name, so `externalAuthors` not `External Authors`.
 
+### Generating a migration script from content model
+
+It's possible to make changes in Contentful UI content type, adding or changing fields and then generate the migration.
+
+To do that you need to follow these steps below:
+
+1. Create an empty migration with the command from the section above
+
+   ```sh
+   yarn workspace @asap-hub/contentful ctf-migrate create <name> -c <content_type>
+   ```
+
+   :warning: Copy the path of the generated file, you will use that in step 5.
+
+2. Add the generated file, create a commit and push to remote and create a draft PR. In the pipeline there's a step called `create-environment/crn-contentful` that will create a PR env in Contentful if there's room for it (max of 6 envs).
+3. Access the PR env and make the changes you want to.
+4. In `.env` fill `CONTENTFUL_SPACE_ID`, `CONTENTFUL_ACCESS_TOKEN` and `CONTENTFUL_ENV_ID`. This field `CONTENTFUL_ENV_ID` should be the PR env that was created and you made changes to it in step 3, something like `crn-2227`.
```

Comment:
I'm not sure what this part of README is about, where you trying to describe what a local development should look like?

Reply:
(I'm replying the comments above in this one)

Not only local, it would be like when we change schema in squidex and sync it with local. The idea is to use ` yarn workspace @asap-hub/contentful ctf-migrate create (...)` to generate only the empty file as the `create` command generates an empty file

> create
Creates an empty time stamped file in the content-type's migrations folder.
https://github.com/deluan/contentful-migrate#create
 
Then from the empty file we can manually type all the fields we want to add and then run something like `yarn workspace @asap-hub/contentful ctf-migrate up -e crn-2227 -d` **or** we can use the generate to sync with the changes we made in the UI in the PR env. So this section is to describe how to do that :)

Reply:
So the issue is that if you generate an empty migration and commit it and push - this migration will run on your PR branch and it will only run once. So you cant really add anything to it later unless you remember to delete it (ie modify migration state) on the dedicated environment. 

I think you should wait until this https://asaphub.atlassian.net/browse/CRN-1121 is complete to suggest any changes to the development and PR dev flow

Reply:
ok! 👍 I removed the instructions to run the generate and the script in package json